crates/kubelet/src/node.rs

```diff
@@ -389,3 +383,120 @@ fn lease_spec_definition(node_name: &str) -> serde_json::Value {
         }
     )
 }
+
+/// Defines the labels that will be applied to this node
+///
+/// Default values and passed node-labels arguments are injected by config.
+fn node_labels_definition(arch: &str, config: &Config) -> HashMap<String, String> {
```

Comment:
Are the node labels always known at compile time? Is there any reason why the hashmap can't be `HasMap<&'static str, &'static str>`?

Reply:
They aren't always known. There are three types of labels as I understand it.  Some A)are static and know at compile type, some have a static key, but a variable value (depend on the arch parameter), and some are supplied by the user. User supplied values are loaded into Config.node_labels which is a HashMap<String, String>. I think that implies that no, it can't be `HashMap<&'static str, &'static str>`.  Sound right?

Reply:
Yes that's correct.